### ISSUE TITLE
Not able to load world with mod enabled anymore

### ISSUE THREAD

**Issue by Craluminum2413** [copygirl/CarryCapacity#14]
**Game version**: 1.16.4-rc.2
**Mod version**: v0.6.5-rc.1
**World**: https://we.tl/t-zk2bJm08DR
### Description
When I load my world with the mod, it crashes. When I disabled it, the world loads fine.
I played on 1.16.4-rc.2 with v0.6.5-rc.1 version just fine, so I don't know what the problem.
**I have storage vessel filled with items on the back.**
### Logs
```
Running on 64 bit Windows with 16 GB RAM 
Game Version: v1.16.4-rc.2 (Unstable)
Loaded Mods: ancienttools@1.3.6, animalcages@2.0.5, tonwexp@1.0.9, civbooks@1.3.2, composter@1.0.2, cottagewindows@1.0.1, dragonflies@1.1.2, ebcc@1.0.0, fancydoors@1.1.0, gwb@1.0.1, glassmaking@1.1.0, gravelsandfix@1.0.2, midnight-texturepack@1.1.0, leadroof@1.1.0, notradermoneylimit@1.0.0, oxygennotincluded@0.2.3, pieontheshelf@1.0.0, peifme@1.0.0, potatoes@1.0.3, potontheshelf@1.0.0, qptech@1.7.1, rackablefirestarter@1.0.0, soilbrick@1.0.1, usejugtocraftdough@1.0.0, viescraftmachines@1.6.0, game@1.16.4-rc.2, carrycapacity@0.6.5-rc.1, extraoverlays@1.1.0, fancyplanters@1.0.2, farmlanddropssoil@1.4.0, fixbadrecipes@1.0.1, fixhandbookclutter@1.0.2, helvehammerext@1.2.0, medievalexpansion@3.7.1, moreanimals@1.0.2, morepiles@1.2.0, petai@1.3.5, pei@1.2.1, peifm@1.0.2, petool@1.0.0, playercorpse@1.3.1, prospectorinfo@3.1.2, ruinvariants@1.0.0, stepup@1.2.0, ssppvv@1.0.2, stonequarry@1.6.3, useplanksinpitkiln@1.0.0, vanvar@1.1.2, creative@1.16.4-rc.2, survival@1.16.4-rc.2, woodbarrels@1.0.1, woodchests@1.0.2, workbenchexpansion@1.4.1, xrowboat@0.0.3, campaigncartographer@2.1.0-pre.1, bullseye@2.2.4, feverstonehorses@1.2.5, instantfirepits@3.0.0, wildcraft@1.1.5
02/19/2022 17:05:49: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at CarryCapacity.Common.CarryHandler.OnLockSlotsMessage(LockSlotsMessage message)
   at Vintagestory.Client.NoObf.NetworkChannel.<>c__DisplayClass12_0`1.<SetMessageHandler>b__0(_2otdrmfSKfjlCdVIjRYcVy8uo7U p)
   at Vintagestory.Client.NoObf.NetworkAPI.HandleCustomPacket(_yyyo0JT9wi3btNTUXP38bgDMMkY packet)
   at Vintagestory.Client.NoObf.NetworkAPI.HandleChannelsPacket(_yyyo0JT9wi3btNTUXP38bgDMMkY packet)
   at Vintagestory.Client.NoObf.ClientMain.ExecuteMainThreadTasks(Single deltaTime)
   at _fxfI2UE0ayN90kkW3TlPfqCqByE._9cfcId0IFdBkNhMH5h3zKScp7JtA(Single )
   at _bjnBWlJotnHiR69aw1flHb4F023._U6WbyQ4am861wMSz04lHr6bdWYyA(Single )
   at _bjnBWlJotnHiR69aw1flHb4F023._PeYLgU97COZFmBgbuLznjIwJUOS(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 476
   at OpenTK.GameWindow.DispatchRenderFrame() in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 452
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 375
   at _As7FGAmUC8gaKIkRNk2zuk1jlJ._zleiWFUbDyAE8J9zHf00bJUUlQr(_M5s9po0jOCy75F2Bz2DU6XdfcDb , String[] )
   at _f8y3Fmmhb3dxIjBj0LM71AeVhBG._zleiWFUbDyAE8J9zHf00bJUUlQr(ThreadStart )
```

**Comment by Craluminum2413** [comment]
Nevermind the world loads fine with 0.6.4 version, just not with 0.6.5-rc.1

**Comment by Craluminum2413** [comment]
So I figured out the problem was in the storage vessel that I had on my back, it seems like 0.6.5-rc.1 cannot handle that after exiting the world

**Comment by copygirl** [comment]
Apparently the cause is the client world's `Player` property not being set when the `LockSlotsMessage` is received. I've tried moving the message sending to `PlayerJoin` instead, but it had the same effect. I can add a tick of delay or so and it works, but at the moment that seems like a dirty workaround, so I'm asking around to see if there's a proper way to send messages after the player joins where ... the player is actually set, you know?

**edit:** The `PlayerNowPlaying` event works for me.

**Comment by copygirl** [comment]
Should be fixed in [v0.6.5-rc.2](https://github.com/copygirl/CarryCapacity/releases/tag/v0.6.5-rc.2).
